test/test_nn.py

```diff
@@ -12987,32 +12987,6 @@ def test_LayerNorm_general(self, device):
         if self.device_type == 'cuda':
             self._test_LayerNorm_cuda_half(device)
 
-    @onlyOnCPUAndCUDA
-    def test_LayerNorm_numeric(self, device):
```

Comment:
@krshrimali please restore this test. It is testing numerics for large inputs, and it is added on purpose. Added OpInfo does not cover this (and it should not, OpInfos are not intended for testing interesting numerics of individual operations).

Reply:
Oops, sorry for not noticing this! I'll put this back in (and cc' you and krshrimali as reviewers)

Reply:
https://github.com/pytorch/pytorch/pull/64385

Reply:
Apologies for not noticing this, thanks for pointing this @ngimel, and thanks @zou3519 for the quick PR. :)